Rakefile

```diff
@@ -87,295 +87,3 @@ Gemfile:
 EOM
   end
 end
-
-PROJECT_ROOT = File.dirname(__FILE__)
-Dir.chdir(PROJECT_ROOT) # Ensure all paths expand relative to this Rakefile.
-
-MODULE_METADATA = JSON.parse(File.read('metadata.json'))
-
-# Set to a tagged version or other git ref.
-PUPPETSERVER_VERSION = if ENV.has_key?('PUPPETSERVER_VERSION')
```

Comment:
We need to keep this puppetserver stuff at least until the end of the year incase we need to rebuild it for some reason. 2018.x goes EOL in Jan.

Everything in the CI and TEST namespace are good to rm though.